embedded-graphics/src/mock_display.rs

```diff
@@ -201,10 +206,24 @@ where
     }
 
     /// Changes the color of a pixel.
-    pub fn set_pixel(&mut self, p: Point, color: Option<C>) {
-        let Point { x, y } = p;
+    pub fn set_pixel(&mut self, point: Point, color: Option<C>) {
```

Comment:
`set_pixel` did intentionally not check for overdraw. It's supposed to be a lower level access than using the `DrawTarget`.

Reply:
This and some of the other `MockDisplay` weirdness was a bad rebase on my part I think. Anyway, the bounds checking has been moved into `MockDisplay`'s `DrawTarget` impl which I think is the correct place?

Reply:
Yes, looks good now.